tests/test_api_log_injection.py

```
@@ -0,0 +1,60 @@
+            mock_response.iter_content.return_value = [b'{"data": []}']
+
+            session_instance = mock_session.return_value
+            session_instance.request.return_value = mock_response
+            client.session = session_instance
+
+            # Call _make_request directly
+            try:
+                client._make_request(endpoint=malicious_endpoint)
+            except Exception as e:
+                # We don't expect an exception here, but if one occurs, catch it
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)